PubnativeLite/PubnativeLite/MRAID/PNLiteMRAIDModalViewController.m

```diff
@@ -94,6 +94,7 @@ - (void)viewWillAppear:(BOOL)animated {
     [HyBidLogger debugLogFromClass:NSStringFromClass([self class]) fromMethod:NSStringFromSelector(_cmd) withMessage:[NSString stringWithFormat:@"%@ %@", [self.class description], NSStringFromSelector(_cmd)]];
 
     isStatusBarHidden = [[UIApplication sharedApplication] isStatusBarHidden];
+    self.view.backgroundColor = [UIColor blackColor];
```

Comment:
Was the color supposed to be black or white? We did it white on Android but I can't remember exactly what PMs wanted.

Reply:
@eros902002 I made it black based on this ticket: https://pubnativegmbh.atlassian.net/browse/OMI-457

Reply:
Cool. Sorry. My bad then.

Reply:
No worries. I forgot what did i have for the lunch most of the time. 😂